### PR TITLE
removed unused css

### PR DESCRIPTION
**Pull request by brianteeman**
I can't find any use of the classes in this additional css file - so lets kill it.

code review

### CODE REVIEW

**Comment by Fedik** [comment]
This also need to remove then:
https://github.com/joomla/joomla-cms/blob/78f01dff40a5684aa872976e572d9c5262563ccb/build/media_source/com_menus/joomla.asset.json#L59-L63

**Comment by Fedik** [comment]
I have tested this item :white_check_mark: successfully on 507279a5f8c989b94f4ccf2d7c97a4efaec102f6<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38353">issues.joomla.org/tracker/joomla-cms/38353</a>.</sub>

**Comment by RickR2H** [comment]
I have tested this item :white_check_mark: successfully on 507279a5f8c989b94f4ccf2d7c97a4efaec102f6<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38353">issues.joomla.org/tracker/joomla-cms/38353</a>.</sub>

**Comment by RickR2H** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38353">issues.joomla.org/tracker/joomla-cms/38353</a>.</sub>

**Comment by fancyFranci** [comment]
I really love to remove unused stuff but I'm moving this to 4.3. I want to be sure it doesn't break someones page on 4.2.

**Comment by brianteeman** [comment]
Very sad face. Just passing the buck

**Comment by obuisard** [comment]
Thank you Brian @brianteeman!

File: administrator/components/com_admin/script.php [hunk 1]
```diff
@@ -6440,6 +6440,7 @@ public function deleteUnexistingFiles($dryRun = false, $suppressOutput = false)
             '/plugins/task/checkfiles/checkfiles.php',
             '/plugins/task/demotasks/demotasks.php',
             // From 4.2.0-rc1 to 4.2.0
+            '/build/media_source/com_menus/css/admin-item-edit_modules.css',
```
Comment (richard67) [inline administrator/components/com_admin/script.php:6443]: This change is wrong. We do not remove things from `/build/media_source` on update!. We remove things from `/media` only. And it is in the wrong section `// From 4.2.0-rc1 to 4.2.0` of the list. I will make a PR to fix both.

Comment (richard67) [reply administrator/components/com_admin/script.php:6443]: PR is #39101 .

**Pull request merged**